deployment/docker/README.md

```diff
@@ -0,0 +1,56 @@
+![Markdown](https://img.shields.io/badge/Make-4.3-red)
+[![Markdown](https://img.shields.io/badge/Golang-1.18-green)](https://go.dev/dl/)
+[![Markdown](https://img.shields.io/badge/Docker-23.0.5-blue)](https://docs.docker.com/get-docker/)
+[![Markdown](https://img.shields.io/badge/Docker%20Compose-1.29.2-blue)](https://github.com/docker/compose)
+![Markdown](https://img.shields.io/badge/jq-1.6-red)
+
+## Minimum System Requirements
+
+- 1 TB of free disk space, accessible at a minimum read/write speed of 100 MB/s.
+- 4 cores of CPU and 8 GB of memory (RAM).
+- A broadband Internet connection with upload/download speeds of at least 1 MB/s.
+
+## Setting up a working envrionment
+
+| Please check the [greenfield repo](https://github.com/bnb-chain/greenfield) for information on the testnet, including the correct version of the binaries to use and details about the config file. |
```

Comment:
Please check the [Release. Note](https://greenfield.bnbchain.org/docs/release-note/) for information on the testnet, and find details in [Greenfield Repo](https://github.com/bnb-chain/greenfield), ....

Reply:
fixed in the new commit